lib/reline/io/ansi.rb

```diff
@@ -347,6 +347,13 @@ def clear_screen
 
   def set_winch_handler(&handler)
     @old_winch_handler = Signal.trap('WINCH', &handler)
+    @old_cont_handler = Signal.trap('CONT') do
+      @input.raw!(intr: true) if @input.tty?
+      # Rerender the screen. Note that screen size might be changed while suspended.
+      handler.call
```

Comment:
📝 Set `@resized` to `true` using `set_signal_handlers`, and rerender with `handle_resized`.